usr/include/pkcs11types.h

```diff
@@ -585,6 +585,8 @@ typedef CK_ULONG CK_ATTRIBUTE_TYPE;
  * opaque attribute. Support has been added to use this attribute
  * in key objects only. */
 #define CKA_IBM_OPAQUE         CKA_VENDOR_DEFINED + 1
+#define CKA_IBM_OPAQUE_REENC   CKA_VENDOR_DEFINED + 3
+#define CKA_IBM_OPAQUE_OLD     CKA_VENDOR_DEFINED + 4
```

Comment:
Please add the commit message as a documentation in the code:
<pre>
Secure key tokens store the secure key blob in attribute CKA_IBM_OPAQUE
in their key object. During an HSM master key change, the secure key blob
is being re-enciphered with the new master key. This re-enciphered secure
key blob is stored in attribute CKA_IBM_OPAQUE_REENC while the HSM master
key change operation is active.

When a HSM master key change operation is finalized, the secure key blob
enciphered with the old master key is backed up into attribute CKA_IBM_OPAQUE_OLD,
and the re-enciphered secure key blob from CKA_IBM_OPAQUE_REENC becomes
the current one in CKA_IBM_OPAQUE.
</pre>